src/__tests__/Navbar.test.js

```diff
@@ -0,0 +1,14 @@
+import { MemoryRouter } from 'react-router-dom';
```

Comment:
Let's create a directory for each component inside the test folder. 
So that you have all Navbar test-related things inside one folder and things don't get messy.